catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlQueryResponse.java

```diff
@@ -57,6 +59,14 @@ public CqlQueryResponse(String id, QueryRequest request, QueryResponse queryResp
 
         status = new Status(queryResponse, source, elapsedTime);
 
+        AtomicBoolean logOnceState = new AtomicBoolean(false);
+        Consumer<String> logOnce = (str) -> {
+            if (logOnceState.compareAndSet(false, true)) {
+                logOnceState.set(true);
```

Comment:
❓ You've already set the atomic to `true` in your condition. Why are you resetting it here?